docs/source/concepts/FittingMinimizersComparisonDetailed.rst

```diff
@@ -80,14 +88,24 @@ weighted least squares as cost function
 Run time for individual NIST problems, "lower" difficulty
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. include:: minimizers_comparison/v3.7.0/comparison_unweighted_v3.7_runtime_nist_lower.txt
+.. include:: minimizers_comparison/v3.8.0/comparison_unweighted_v3.8_runtime_nist_lower.txt
 
 Run time for individual NIST problems, "average" difficulty
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. include:: minimizers_comparison/v3.7.0/comparison_unweighted_v3.7_runtime_nist_average.txt
+.. include:: minimizers_comparison/v3.8.0/comparison_unweighted_v3.8_runtime_nist_average.txt
 
 Run time for individual NIST problems, "higher" difficulty
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. include:: minimizers_comparison/v3.7.0/comparison_unweighted_v3.7_runtime_nist_higher.txt
+.. include:: minimizers_comparison/v3.8.0/comparison_unweighted_v3.8_runtime_nist_higher.txt
+
+Run time for individual Neutron Data problems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. include:: minimizers_comparison/v3.8.0/comparison_unweighted_v3.8_runtime_neutron_data.txt
```

Comment:
I see the "Neutron data" header, then the "CUTEst" results table - the table/header in the middle are missing.